doc-www/README.txt

```diff
@@ -0,0 +1,71 @@
+doc-www README.txt
+====================
+
+This doc-www/ directory contains documentation about the `ompi-www` website itself. This stuff
+is for developers who wish to work on maintaining the website, as opposed to people who just
+want to learn about Open MPI.
+
+
+# Hosting Requirements
+
+This section should contain doco describing what sort of web server and libraries etc. you 
+need to run the ompi-www web site.
+
+I'm just guessing here:
+
+* A web server (uh, Apache, I guess?)
+* PHP
+* ...???
+
+TODO: Finish me.
+
+
+# Running the Site Locally
+
+This section should explain how you can run a locally-hosted copy of the web site from your
+working copy of the `ompi-www` Git repo, so you can test out your local changes before 
+submitting them as a PR.
+
+TODO: Fill in this section.
```

Comment:
Should just be `git clone` -- it should run fine from there.

That being said, there may be sections that don't work.  Let's see...

* Downloads should all be coming from S3, so they're not part of the repo.  So they should work.
* Archives of old stuff (e.g., mail archives) are all in the git repo (and are static).  So they should work, too.

Hmm.  I guess I'm not thinking of anything offhand that *won't* work with a local git clone... but something is nagging me that there *is* something that is known not to work.  Hmm.  Can't think of what it is.  ☹️